src/main/java/org/baeldung/security/LoggedUser.java

```diff
@@ -0,0 +1,50 @@
+package org.baeldung.security;
+
+import java.util.List;
+
+import javax.servlet.http.HttpSessionBindingEvent;
+import javax.servlet.http.HttpSessionBindingListener;
+import javax.servlet.http.HttpSessionEvent;
+import javax.servlet.http.HttpSessionListener;
+
+import org.baeldung.web.dto.LoggedUserDto;
```

Comment:
Did you push your changes after second commit? I don't see the change in the pull request.